ur_robot_driver/launch/ur_control.launch.py

```diff
@@ -530,7 +530,7 @@ def generate_launch_description():
     declared_arguments.append(
         DeclareLaunchArgument(
             "tool_voltage",
-            default_value="24",
+            default_value="0", # 0 being a conservative value that won't destroy anything
```

Comment:
I think that a space is missing before the comment to make the formatter happy.

Reply:
> Is it the plan to use the control script from the client library as we do in the ROS1 driver?

See #316